packages/appstore/src/modules/traders-hub/index.tsx

```diff
@@ -91,33 +93,17 @@ const TradersHub = () => {
     };
     if (!is_logged_in) return null;
 
-    const EUDisclamer = () => {
-        return (
-            <div className='disclamer'>
-                <Text align='left' className='disclamer-text' size={isMobile() ? 'xxxs' : 'xs'}>
-                    <Localize
-                        i18n_default_text='<0>EU statutory disclaimer</0>: CFDs are complex instruments and come with a high risk of losing money rapidly due to leverage. <0>71% of retail investor accounts lose money when trading CFDs with this provider</0>. You should consider whether you understand how CFDs work and whether you can afford to take the high risk of losing your money.'
-                        components={[<strong key={0} />]}
-                    />
-                </Text>
-            </div>
-        );
-    };
-
-    const contentForOptionandCfdListing = () => {
-        if (is_eu_user && is_real) {
-            return (
-                <div className='traders-hub__main-container'>
-                    <CFDsListing />
-                    <OptionsAndMultipliersListing />
-                </div>
-            );
-        }
-        return (
-            <div className='traders-hub__main-container'>
-                <OptionsAndMultipliersListing />
-                <CFDsListing />
-            </div>
+    const renderOrderedPlatformSections = (is_cfd_visible = true, is_options_and_multipliers_visible = true) => {
+        return is_eu_user ? (
+            <React.Fragment>
+                {is_cfd_visible && <CFDsListing />}
```

Comment:
**NIT** If the order of components is the only change that occurs based on the flag, can they be rearranged using CSS `flex-direction` ?

Reply:
@likhith-deriv thanks. will implement this using css.